test/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "extends": "../tsconfig.json",
+  "files": [
+    "FusionAuthClientTest.ts"
+  ],
+  "compilerOptions": {
+    "types": [
+      "node",
+      "node-fetch",
+      "mocha",
+      "chai"
+    ]
+  }
+}
```

Comment:
This inherits the parent tsconfig and adds the test file + the test framework types (for type resolution)